app/templates/app/detail_contract.html

```diff
@@ -0,0 +1,78 @@
+        </div>
+        <div class="jumbotron">
+            <p class="lead indent"><b>Account name: </b> {{contract.organizer_account_name}}</p>
+            <p class="lead indent"><b>Email: </b> {{contract.organizer_email}}</p>
+            <p class="lead indent"><b>Signed date: </b>{{contract.signed_date}}</p>
+            {% if contract.user_id %}
+                <p class="lead indent"><b>User id: </b> <a href="{{link_to_recoup}}">{{contract.user_id}}</a></p>
+            {% endif %}
+            <p class="lead indent"><b>Salesforce case: </b> <a href="{{contract.link_to_salesforce_case}}">{{contract.salesforce_case_id}}</a></p>
+            {% if events %}
+                <p class="lead indent">
+                    <b>Events:</b>
+                </p>
+                    <div class="margin-event-table">
+                    {% for event in events %}
+                        <table class="table table-borderless">
+                            <td class="lead table-td-event"><a href="{{event.link_to_event}}" target="_blank">{{event.event_name}}</a></td>
+                            <td class="table-td-trash">
+                                <form class="delete-file-form" action="{% url 'events-delete' contract.id event.id %}" method="POST">
+                                    {% csrf_token %}
+                                    <button class="btn btn-link" type="submit">
+                                        <i class="far fa-trash-alt"></i>
+                                    </button>
+                                </form>
+                            </td>
+                        </table>
+                    {% endfor %}
+                    </div>
+            {% else %}
+                <p class="lead indent"><i>There are no events for this contract yet.</i></p>
+            {% endif%}
+            <div class="lead indent-form">
+                <form action="{% url 'events-create' contract.id %}" method="POST">{% csrf_token %}
+                    <div class="form-group row">
+                        <div class="col-sm-3">
+                            <input id="event_name" type="text" class="form-control" name="Event Name" value="" placeholder="Event name">
+                        </div>
+                        <div class="col-sm-3">
+                            <input id="event_id" class="form-control" type="text" name="Event id" value="" placeholder="Event id">
+                        </div>
+                        <input type='submit' class="btn btn-outline-primary" value='Save'/>
+                    </div>
+                </form>
+            </div>
+        </div>
+        <a href="{% url 'contracts' %}" role="button" class="btn btn-outline-secondary"><i class="far fa-arrow-alt-circle-left"></i>  Back</a>
+    </div>
+    <script>
+        $('.delete-file-form').submit(e => {
+            e.preventDefault();
+            if (confirm('Confirm delete for this event')) e.target.submit();
+        });
```

Comment:
Sounds like that you start this PR from a old branch...

Reply:
Yes, i forgot delete this code.